src/components/ask/ask.scss

```diff
@@ -0,0 +1,51 @@
+@import 'src/utils/style-utils.scss';
```

Comment:
Get rid of everything in this file (except the first line import). You don't need any of this for this component.